src/motors/motor_init.c

```diff
@@ -147,8 +141,8 @@ void InitIC(short motIdx) {
             // Initialize Capture Module
             IC1CONbits.ICM = IC_DISABLE; // Disable Input Capture 1 module
             IC1CONbits.ICTMR = 1; // Select Timer2 as the IC1 Time base
-            IC1CONbits.ICI = 0b01; // Interrupt on every second capture event
-            IC1CONbits.ICM = IcMode[0]; // Generate capture event on every Rising edge
+            IC1CONbits.ICI = 0b00; // Interrupt on every second capture event
```

Comment:
In [IC reference dsPIC33f](http://ww1.microchip.com/downloads/en/DeviceDoc/70198D.pdf) in page 4 show:

> **ICM<2:0>**: Input Capture x Mode Select bits
> ...
> 001 = Capture mode, every edge –  rising and falling (ICI<1:0> bits do not control interrupt generation for this mode)

Is **REQUIRED** to works better to set
``` c
IC1CONbits.ICI = 0b00; // Interrupt on every capture event
```
To send the interrupt every event